definitions/contested/json/ComplexTypes/ComplexTypes.json

```diff
@@ -2583,7 +2583,9 @@
     "ID": "FR_supportDocument",
     "ListElementCode": "supportDocument",
     "FieldType": "Document",
+    "RegularExpression": ".doc,.docx,.pdf,.xls,.xlsx",
     "ElementLabel": "Supporting Document",
+    "HintText": "Please upload a copy of the supporting document as a word, PDF, or Excel document (word/Excel documents will be converted to PDF after submission)",
```

Comment:
I think instances of "word" should be capitalised in the same we that we have Excel.